src/renderer/qss/js/qssServiceButtons.js

```diff
@@ -28,6 +28,14 @@
         attrs: {
             "aria-label": "Settings Panel"
         },
+        listeners: {
+            "onArrowUpPressed.activate": {
```

Comment:
Probably a comment like: "Overriding focusManager `upArrow` behaviour"

Reply:
Technically, this is not overriding. Each button has 2 listeners which are triggered when Arrow Up is pressed - the focus manager and the button activation. It is up to the person defining the QSS to make sure that a button which is not first in its group cannot be activated using the Arrow Up.